pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -211,6 +211,15 @@ var ProtectTimestampInterval = settings.RegisterDurationSetting(
 	settings.PositiveDuration,
 )
 
+// MaxProtectedTimestampAge controls the frequency of protected timestamp record updates
+var MaxProtectedTimestampAge = settings.RegisterDurationSetting(
+	settings.TenantWritable,
+	"changefeed.protect_timestamp.max_age",
+	"fail the changefeed is the protected timestamp age exceeds this threshold; 0 disables expiration",
```

Comment:
```suggestion
	"fail the changefeed if the protected timestamp age exceeds this threshold; 0 disables expiration",
```

Reply:
Done; FYI: I also updated changefeed_stmt warning message to produce better warning when this setting is too low.